spec/item_repository_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'item_repository'
+
+RSpec.describe ItemRepository do
+  # snippet and function call copied and modified from repository class recipe
```

Comment:
I'm not sure if you need this comment for the code, I don't need it when reading. Maybe confusing, if I don't know what the recipe was.